modules/modify-data-object/vendor/github.com/kubevirt/kubevirt-tekton-tasks/modules/sharedtest/testobjects/datasource/datasource.go

```diff
@@ -61,6 +61,11 @@ func (d *TestDataSource) WithSourcePVC(name, namespace string) *TestDataSource {
 	return d
 }
 
+func (d *TestDataSource) WithGenerateName(generateName string) *TestDataSource {
```

Comment:
New helper function in rename PR?

Reply:
this is a leftover from previous PR, when someone forget to run make vendor.